### PR TITLE
[Tests-Only] Acceptance test for password protected public share

### PR DESCRIPTION
**Pull request by HariBhandari07**
## Description

We currently don't have an acceptance test to assign a password to an already existing public share so this PR adds an acceptance test for that.

## Related Issue

-  https://github.com/owncloud/web/issues/4614
## How Has This Been Tested?

- Manual
- CI
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by swoichha — COMMENTED** [review]
LGTM :+1:

File: tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature [hunk 1]
```diff
@@ -871,3 +871,13 @@ Feature: Share by public link
       | lorem.txt           |
       | simple-empty-folder |
       | data.zip            |
+
+  Scenario: assign password to already created public share
```
Comment (Talank) [inline tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature:875]:
```suggestion
  Scenario: file owner can assign a password to already created public share
```

**Review by Talank — COMMENTED** [review]
I think the scenario description can be made more sensible. Other than that, everything LGTM

File: tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature (hunk 1 above)
Comment (HariBhandari07) [reply tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature:875]: The purpose of this scenario is to check that password can be assigned to already existing public share, so IMO no need to change the description.

**Comment by kiranparajuli589** [comment]
lgtm :+1:

**Review by individual-it — APPROVED** [review]

**Pull request merged**